superset-frontend/src/setup/setupFormatters.ts

```diff
@@ -78,6 +79,7 @@ export default function setupFormatters(
       'DURATION_SUB',
       createDurationFormatter({ formatSubMilliseconds: true }),
     )
+    .registerValue('TIME_DURATION', createTimeDurationFormatter())
```

Comment:
Please check the `README.md` in the `number-format` directory for an example how to implement this. I believe this should do it:
```suggestion
    .registerValue('TIME_DURATION', createTimeDurationFormatter({ colonNotation: true }))
```

Reply:
Thank you. This does not the same, it will format it in hh:mm:ss and I would like to create a formatter for hh:mm. However, I can add both, DURATION_COL and DURATION_COL_SHORT. What are your thoughts?